server/src/test/java/org/elasticsearch/snapshots/SnapshotsInProgressSerializationTests.java

```diff
@@ -414,9 +417,27 @@ public void testXContent() throws IOException {
                 null,
                 IndexVersion.current()
             )
-        );
+        )
+            .withUpdatedNodeIdsForRemoval(
```

Comment:
I think we also need a wire serialization change in `createTestInstance` and `mutateInstance` to verify the serialization of the `nodesIdsForRemoval` field?

Reply:
++ see c495f381e1ad1d8ace22b57c3450266cd7ef2659.